### DISCUSSION TITLE
Request for Entry "Updated at" timestamps to be unique per their sites

### DISCUSSION THREAD

**Discussion by ifrimere** [craftcms/cms#8702] [Ideas]
### Description
Currently, if a secondary site's Entry has it's content updated, the primary site's corresponding Entry has its 'Updated at' timestamp modified.  This seems like the wrong behavior, because each site's Entries should have their own unique 'Updated at' timestamps based on when they were actually last updated.

### Steps to reproduce

1. Create a multisite Craft3 site and set EN(US) as default site, and DE as the secondary site.
2. Access the homepage in DE, modify its content and save/publish it.
3. Check the homepage in EN(US) to see when it was last 'updated'.  It's timestamp should not have changed after the DE homepage was updated.

### Additional info

1. For our Translation Plugin, we have some features that we'd like to implement to help our clients be better notified when their translations are 'out of date' due to Source content changes... and hope that they then order more translation work from us.  To best implement these features we are looking to do diffs of 'Updated at' timestamps to see if source content was modified after translations were 'published'.

**Comment by brandonkelly** [comment]
Each time you save an entry, it is saved across all sites. So even if `dateUpdated` returned the site-specific timestamp, it would still be identical to the other sites’ timestamps.

**Reply by hldrm** [reply]
how about an additional field, similar to "postDateSite1", "postDateSite2":
"dateUpdatedSite1", "dateUpdatedSite2"?

Otherwise it is almost impossible to find that information.

Is that maybe coming with Craft 4.0 anyways?

**Comment by ifrimere** [comment]
@brandonkelly - can this ticket be reopened as a new feature request to add a more granular aspect to the saved entry db record that is specific to ‘unique content’ changes on the children Entries per enabled Sites. Without this feature, if you imagine a source English site and 20 target language sites, how could users have an at a glance understanding of target entries that are up to date and those whose translations are out of sync.

**Comment by arnedem** [comment]
@brandonkelly Is there any news regarding this issue? It would be really helpful to know when the entry was updated for each site, as @hldrm and @ifrimere indicated.

**Reply by brandonkelly** [reply]
Sorry, no, per my previous comment. I’m not sure there’s a good way to resolve this considering elements are saved across each of their sites each time they are saved.
